frontend/src/organisms/UserProfileForm.js

```diff
@@ -29,10 +29,10 @@ export function UserProfileForm({ user, updateUserRequest }) {
     username: user.username,
     email: user.email,
     phone: user.phone,
-    street: user.places[0] ? user.places[0].street : undefined,
-    city: user.places[0] ? user.places[0].city : undefined,
-    country: user.places[0] ? user.places[0].country : undefined,
-    zip: user.places[0] ? user.places[0].zip : undefined,
+    street: user.places[0] ? user.places[0].street : "",
```

Comment:
tady myslím ještě říkal , jestli by to nešlo vytáhnout do vlastní proměnné to user.places[0], ale osobně mi to zas až tak nevadí

Reply:
jo a pak to pouzivat jako: 
 promena || "";